my_list.rb

```diff
@@ -0,0 +1,20 @@
+require './my_enumerable'
+
+class MyList
+  def initialize(list)
+    @list = list
+  end
+
+  include MyEnumerable
+
+  def each(&block)
```

Comment:
- [ ] It would be better if you implement each method without using the inbuilt `each` method.  You must implement your own each for this exercise.

![Capture](https://user-images.githubusercontent.com/64173189/162345415-0fa87b6a-9761-4aea-a89f-ff83c30ede78.JPG)